swig/python/GNUmakefile

```diff
@@ -1,7 +1,5 @@
 
 
-include ../../GDALmake.opt
```

Comment:
This file doesn't exist and causes the makefile to error.

Reply:
swig/python/GNUmakefile is going to be removed per https://github.com/OSGeo/gdal/pull/6110

Reply:
Makes sense - removed it.